### PR TITLE
REV-71-signup-page-no-restrictions-on-password-field

### PR DESCRIPTION
**Pull request by IbrahimAldereni-socuimtech**
### Jira URL

https://optimumpartners.atlassian.net/browse/REV-71

### Description

- add restrictions Password field

Screenshot from 2022-02-05 13-50-11

### CODE REVIEW

**Review by mohdabed123 — APPROVED** [review]

File: src/pages/SignUp/sign-up-schema.js [hunk 1]
```diff
@@ -1,3 +1,4 @@
+/* eslint-disable no-useless-escape */
```
Comment (AhmadAlShwaiki) [inline src/pages/SignUp/sign-up-schema.js:1]:
```suggestion
```

File: src/pages/SignUp/sign-up-schema.js [hunk 2]
```diff
@@ -9,6 +10,11 @@ const signUpSchema = yup
       .string()
       .label('Password')
       .required()
+      .min(8, 'password is too short - should be 8 chars minimum')
+      .matches(
+        /^(?=.*[a-z])(?=.*[A-Z])(?=.*[0-9])(?=.*[!@#\$%\^&\*])(?=.{8,})/,
```
Comment (AhmadAlShwaiki) [inline src/pages/SignUp/sign-up-schema.js:15]:
```suggestion
 NameOfWhatThisRegexDoes
```

Comment (IbrahimAldereni-socuimtech) [reply src/pages/SignUp/sign-up-schema.js:15]: under it here a message for what it does

File: src/pages/SignUp/sign-up-schema.js (hunk 1 above)
Comment (IbrahimAldereni-socuimtech) [reply src/pages/SignUp/sign-up-schema.js:1]: must be added for regex

File: src/pages/SignUp/sign-up-schema.js (hunk 2 above)
Comment (AhmadAlShwaiki) [reply src/pages/SignUp/sign-up-schema.js:15]: Please use a constant, we might want to use this regex elsewhere.

File: src/pages/SignUp/sign-up-schema.js (hunk 1 above)
Comment (AhmadAlShwaiki) [reply src/pages/SignUp/sign-up-schema.js:1]: Please update as per comment below

File: src/pages/SignUp/sign-up-schema.js (hunk 2 above)
Comment (IbrahimAldereni-socuimtech) [reply src/pages/SignUp/sign-up-schema.js:15]: where should i put that constant?

**Review by AhmadAlShwaiki — APPROVED** [review]

**Pull request merged**